MAINTAINERSHIP.md

```diff
@@ -21,39 +20,30 @@ Reach out to the IBC product team through susannah@interchain.io to coordinate u
 
 Answer these questions in a requirements doc:
 
-    What problem does this feature solve?
-
-    What are the current solutions or workarounds? 
-
-    Are there other versions or implementations?
-
-    If there are other versions of this feature, why is this solution better?
-
-    What are the use cases?
-
-    Which users have confirmed they will use this? 
+- What problem does this feature solve?
+- What are the current solutions or workarounds?
+- Are there other versions or implementations?
+- If there are other versions of this feature, why is this solution better?
+- What are the use cases?
+- Which users have confirmed they will use this?
+- How urgent is it to implement this feature?
+- How soon after being developed would this be adopted?
+- What is the impact of this feature being adopted?
 
-    How urgent is it to implement this feature?
+Is there a specific need for this feature to be included in the `ibc-go` codebase, rather than in its own module repo?
```

Comment:
Isn't this also a bullet?

Reply:
Yes, good catch! I fixed this one manually myself :)